lib/teleterm/gateway/gateway.go

```diff
@@ -89,14 +78,13 @@ func New(cfg Config) (*Gateway, error) {
 	}
 
 	localProxyConfig := alpn.LocalProxyConfig{
-		InsecureSkipVerify: cfg.Insecure,
-		RemoteProxyAddr:    cfg.WebProxyAddr,
-		Protocols:          []alpncommon.Protocol{protocol},
-		Listener:           listener,
-		ParentContext:      closeContext,
-		SNI:                address.Host(),
```

Comment:
Is it no longer necessary to set `SNI`? Or should we replace setting this field here with `alpn.WithSNI` around line 101?

Reply:
https://github.com/gravitational/teleport/blob/f9657ab662ef235984e33daa4cd1ec0ac36816e8/lib/srv/alpnproxy/local_proxy.go#L128-L135

It's no longer necessary. The local proxy now will set the same thing by default if SNI is not provided. Only in rare cases (kube access) the SNI has to be explicitly set.